Objects.py

```diff
@@ -1,44 +1,50 @@
 class Player:
     players = []
     pairingnr_to_name = {}
-    pairingnumber = 0  
+    pairingnumber = 0
```

Comment:
misschien iets duidelijker het verschil maken tussen 1 Player en alle players. 
Je zou hier ook een pairingnummer aan toe kunnen voegen wat je vult nadat je de loting verricht. Of blijven de spelers over meerdere jaren hetzelfde? 

Anyway, als je over alle spelers zoiets doet, kan je er naar refereren met pairing nummer en naam.
``` 
player_dict = {}
for player in players:
    player_dict[player.name] = player
    player_dict[player.pairing_number] = player
```